src/system.jl

```diff
@@ -186,7 +187,39 @@ function Branch(
         rate_b,
         is_monitored,
         break_points,
-        penalities,
+        penalties,
+        resistance,
+        reactance,
+        is_transformer,
+        tap,
+        angle
+    )
+end
+
+function Branch(;
+    name,
+    to_bus,
+    from_bus,
+    rate_a,
+    rate_b,
+    is_monitored,
+    break_points,
+    penalties,
+    resistance,
+    reactance,
```

Comment:
should `resistance` and `reactance` have defaults  too?

Reply:
I don't think so, can't see any specific reason to have default values for these